src/components/CldImage/CldImage.vue

```diff
@@ -168,14 +179,14 @@ export default {
       });
 
       return {
-          ...this.nonCldAttrs,
-          ...htmlAttrs,
-          src
+        ...this.nonCldAttrs,
+        ...htmlAttrs,
+        src
       };
     },
     shouldMeasureSize() {
-      return !this.responsive;
+      return this.responsive !== false;
```

Comment:
is there any difference or advantage in writing like this instead of `!this.responsive`?

Reply:
<cld-image responsive /> produces empty string and it should mean 'default value', yet here was interpreted as no value